muhireheir/mybrand-1
# add new blog

> #### What does this PR do?

> * create blogs schema
> * add new blog
> #### Description of Task to be completed
> * create/ add new blog post
> #### How should this be manually tested?
>  * clone the repository
> * change to ft-createBlog3
> * open  terminal  in server folder
> * run  the  following   codes `npm start` and check the port which this project runs on
> * open postman
> * i  url enter the following adress `localhost:{port}/blogs`

![newblog](https://user-images.githubusercontent.com/26524106/94141002-699c5e80-fe6c-11ea-9388-c14227bcb71e.png)

> * send request with post method

> #### Any background context you want to provide?
> * add blogs 
> #### What are the relevant Trello story?
>  * https://trello.com/c/MiU9yKIz
> #### Screenshots (if appropriate)
